ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -79,7 +79,7 @@
     env: "{{ kafka_env_vars }}"
     ports:
       - "{{ kafka.port + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.port + groups['kafkas'].index(inventory_hostname) }}"
-      - "{{ kafka.advertisedPort  + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}"
+      - "{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}"
```

Comment:
Would normally ask for a separate 'typo' PR for this, but seems harmless enough to include it with other ansible changes.